compiler/optimizer/RegisterCandidate.hpp

```diff
@@ -63,25 +62,13 @@ class GlobalSet
    bool isEmpty() { return _refAutosPerBlock.empty(); }
    void makeEmpty() { _refAutosPerBlock.clear(); }
 
-   class Set
-      {
-      public:
-      TR_ALLOC(TR_Memory::RegisterCandidates);
-      Set(TR::Region &region):_refs(region){}
-      bool get(uint32_t refId) { return _refs.get(refId); }
-      void set(uint32_t refId) { _refs.set(refId); }
-      void print(TR::Compilation * comp);
-      private:
-      TR_BitVector _refs;
-      };
-
```

Comment:
"very thin wrapper" for bit vector sounds right :) Not clear if there was some cleaner abstraction kind of reason why this was done but it is just a memory consumer the way it is at least.